src/Components/Admin/admin.js

```diff
@@ -139,9 +139,14 @@ export default function Admin() {
         setAnchorEl(null);
     };
 
-    // TODO: POST room details to backend
-    const handleCreateRoomSubmit = () => {
+    const handleCreateRoomSubmit = async ({
+        name,
+        description,
+        friendly_name,
+    }) => {
         setCreateModalOpen(false);
```

Comment:
Is error handling and character restrictions (on friendly name for example) going to be a future PR?

Reply:
Yes, not a priority right now.